nodewatcher/modules/events/sinks/db_sink/templates/events/list.html

```diff
@@ -1,29 +1,30 @@
 {% extends "base.html" %}
 
 {% contextblock %}
-    {% load future i18n sekizai_tags html_tags url_tags %}
+    {% load i18n sekizai_tags html_tags url_tags %}
 
     {% add_data "css_data" "datatables/css/style.css" %}
     {% add_data "js_data" "datastream/underscore/underscore.js" %}
     {% add_data "js_data" "datastream/moment/moment.js" %}
     {% add_data "js_data" "datatables/js/jquery.dataTables.js" %}
     {% add_data "js_data" "datatables/js/jquery.dataTables.bootstrap.js" %}
     {% add_data "js_data" "datatables/js/jquery.dataTables.fixedHeader.js" %}
-    {% add_data "js_data" "tastypie/js/code.js" %}
+    {% add_data "js_data" "api/js/base.js" %}
+    {% add_data "js_data" "api/js/registry.js" %}
     {% add_data "js_data" "events/list/js/code.js" %}
 
     {% trans "Events" as title %}
 {% endcontextblock %}
 
 {% block altlinks %}
-    {% url "api:api_dispatch_list" "v1" "event" as api_url %}
+    {% url "apiv2:serializednodeevent-list" as api_url %}
```

Comment:
Yes, so should we change it to `api` and the old one to `apiv1` for now?

I would then migrate also datastream (after we discuss how to do it, find me on chat), and then I would release 3.0 finally and merge `development` branch into `master` and start having releases.